syntaxes/haskell.YAML-tmLanguage

```diff
@@ -359,13 +360,15 @@ patterns:
     patterns:
       - include: '#type_signature'
     name: meta.type-declaration.haskell
-  - comment: >
+  - match: >-
+      ((?:[\p{Lu}\p{Lt}][\p{Ll}_\p{Lu}\p{Lt}\p{Nd}'']*\.)*)([\p{S}\p{P}&&[^(),;\[\]`{}_"']]+)
```

Comment:
Do we have to add this to type operators as well?

Reply:
Yes, you can qualify symbolic operators too. See the test case `T0031`, all the qualifications therein are valid Haskell.